### ISSUE TITLE
COBOL: INSPECT import (#631) doesn't really work

### ISSUE THREAD

**Issue by codemanyak** [fesch/Structorizer.Desktop#665]
The import of INSPECT statements fromCOBOL sources does not actually work in the delivered product (release 3.29) because I had forgotten to modify the make script such that the resource diagrams aren't copied to the bin directory.

**Comment by codemanyak** [comment]
Will be fixed with version 3.29-01.

**Comment by codemanyak** [comment]
The files are now in place but there is still a problem retrieving them as resources in the delivered product (not so in the delvelopment environment).
Windows:
```
java.nio.file.InvalidPathException: Illegal char &lt;:&gt; at index 4: file:\C:\tools\Structorizer\Structorizer.app\Contents\Java\Structorizer.jar!\lu\fisch\structorizer\parsers\INSPECT_CONVERTING-5.nsd
```
Linux:
```
java.io.FileNotFoundException: /usr/mnt/file:/usr/mnt/Structorizer/Structorizer.app/Contents/Java/Structorizer.jar!/lu/fisch/structorizer/parsers/INSPECT_REPLACING-6.nsd
```
